kubernetes/kubernetes/src/main/java/org/arquillian/cube/kubernetes/impl/locator/DefaultKubernetesResouceLocator.java

```diff
@@ -7,24 +7,23 @@
 public class DefaultKubernetesResouceLocator implements KubernetesResourceLocator {
 
     private static final String ROOT = "/";
-    private static final String RESOURCE_NAME = "kubernetes";
+    private static final String[] RESOURCE_NAMES = new String[] { "kubernetes", "META-INF/fabric8/kubernetes" };
     private static final String[] ALLOWED_SUFFIXES = {".json", ".yml", ".yaml"};
 
-
     @Override
     public URL locate() {
         for (String suffix : ALLOWED_SUFFIXES) {
-            URL candidate = getResource(RESOURCE_NAME + suffix);
-            if (candidate != null) {
-                return candidate;
+            for (String resource : RESOURCE_NAMES) {
+                URL candidate = getResource(resource + suffix);
```

Comment:
We give priority to `kubernetes` location right? I think this is how is should be, but just to be sure.